tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/device_info.py

```diff
@@ -137,75 +135,80 @@ def confirm_reset(self, response=True):
             self.marionette.find_element(*self._reset_cancel_locator).tap()
             Wait(self.marionette).until(expected.element_displayed(self.screen_element))
 
-    class MoreInfo(Base):
+    class MoreInfo(PageRegion):
 
-        _more_information_page_locator = (By.ID, 'about-moreInfo')
-        _os_version_locator = (By.CSS_SELECTOR, '#about-moreInfo small[data-name="deviceinfo.os"]')
-        _hardware_revision_locator = (By.CSS_SELECTOR, '#about-moreInfo small[data-name="deviceinfo.hardware"]')
-        _mac_address_locator = (By.CSS_SELECTOR, '#about-moreInfo small[data-name="deviceinfo.mac"]')
+        _root_locator = (By.ID, 'about-moreInfo')
+        _os_version_locator = (By.CSS_SELECTOR, 'small[data-name="deviceinfo.os"]')
+        _hardware_revision_locator = (By.CSS_SELECTOR, 'small[data-name="deviceinfo.hardware"]')
+        _mac_address_locator = (By.CSS_SELECTOR, 'small[data-name="deviceinfo.mac"]')
         _imei1_locator = (By.CSS_SELECTOR, '.deviceInfo-imeis span[data-slot="0"]')
         _imei2_locator = (By.CSS_SELECTOR, '.deviceInfo-imeis span[data-slot="1"]')
         _iccid_locator = (By.CSS_SELECTOR, '.deviceInfo-iccids')
-        _platform_version_locator = (By.CSS_SELECTOR, '#about-moreInfo small[data-name="deviceinfo.platform_version"]')
-        _build_id_locator = (By.CSS_SELECTOR, '#about-moreInfo small[data-name="deviceinfo.platform_build_id"]')
-        _build_number_locator = (By.CSS_SELECTOR, '#about-moreInfo small[data-name="deviceinfo.build_number"]')
-        _update_channel_locator = (By.CSS_SELECTOR, '#about-moreInfo small[data-name="app.update.channel"]')
+        _platform_version_locator = (By.CSS_SELECTOR, 'small[data-name="deviceinfo.platform_version"]')
+        _build_id_locator = (By.CSS_SELECTOR, 'small[data-name="deviceinfo.platform_build_id"]')
+        _build_number_locator = (By.CSS_SELECTOR, 'small[data-name="deviceinfo.build_number"]')
+        _update_channel_locator = (By.CSS_SELECTOR, 'small[data-name="app.update.channel"]')
         _git_commit_timestamp_locator = (By.CSS_SELECTOR, '.gaia-commit-date')
         _git_commit_hash_locator = (By.CSS_SELECTOR, '.gaia-commit-hash')
 
         def __init__(self, marionette):
-            Base.__init__(self, marionette)
-            Wait(self.marionette).until(
-                expected.element_displayed(*self._os_version_locator))
+            root = marionette.find_element(*self._root_locator)
+            PageRegion.__init__(self, marionette, root)
+            Wait(self.marionette).until(expected.element_displayed(*self._os_version_locator))
 
         @property
         def screen_element(self):
```

Comment:
This is used when the page needs to be scrolled, and when returning back from its subpages via the back button, I check whether this element is visible